README.md

```diff
@@ -1,24 +1,61 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This project is The Ruby on Rails capstone project which is about building a mobile web application where the user can manage his budget: 
+he can have a list of transactions associated with a category, so that he can see how much money you spent and on what.
 
-Things you may want to cover:
+This Ruby on Rails application allows the user to:
 
-* Ruby version
+* register and log in, so that the data is private to them.
+* introduce new transactions associated with a category.
+* see the money spent on each category.
 
-* System dependencies
+## [Live Demo](https://afternoon-anchorage-61931.herokuapp.com/)
 
-* Configuration
```

Comment:
- [ ] In recorded video please address a piece of code/feature you liked about the project.

Reply:
The delete option for the transaction and category which is not mentioned in the requirements but I did it